dockers/actions-plan-preview/planpreview.go

```diff
@@ -257,3 +264,22 @@ func groupApplicationResults(apps []ApplicationResult) (changes, pipelines, quic
 	}
 	return
 }
+
+func getActionsURL() string {
```

Comment:
nits, should name it as `makeActionLogURL` to unify with other, but it's up to you 😄

Reply:
Thank you! I fixed.